src/tokens.js

```diff
@@ -22,6 +23,13 @@ import { HATHOR_TOKEN_CONFIG, TOKEN_CREATION_MASK, TOKEN_MINT_MASK, TOKEN_MELT_M
  */
 
 const tokens = {
+
+  /*
+   * This is a default value for the token deposit percentage. We should update
+   * using the version API
+   */
+  depositPercentage: 0.01,
```

Comment:
We don't raise an exception when any of the other values from version API is not set. Seems it would be an inconsistent behavior to do this just for `depositPercentage`. Either we change for all or none. If we decide to do it, we'd probably have to update more places than the scope of this PR.

The way things are, I prefer to have a default. If we don't get the value from server and the percentage is wrong, the mint tx won't be accepted anyway.

Reply:
I don't think it's a problem to do only for `depositPercentage ` right now. I would prefer doing for this value only and then from now on we always do.

@msbrogli had a big problem because the API values were not set, at least this would have given him a hint.

Reply:
Opened #21 for dealing with constants not set. When that is merged to dev, I'll update this branch and use the same logic for `depositPercentage `.

Reply:
Changes made. Please review again.